frontend/src/stores/logs.store.ts

```diff
@@ -5,17 +5,21 @@ import { ref } from 'vue'
 export const useLogsStore = defineStore('logs', () => {
   // State
   const logs = ref<Log[]>([])
-  const limit = ref(100)
+  const total = ref(50)
+  const limit = ref(20)
```

Comment:
It was asked to get 100 records by default in Jira, but I also prefer with 20.

Reply:
ok, I had not been aware of this, yes, Jira says by default 100 but via config, let's check with the clients and then decide whether to change the value or not